packages/sources/synthetix-debt-pool/test/integration/adapter.test.ts

```diff
@@ -24,17 +26,24 @@ jest.mock('ethers', () => ({
 
 let oldEnv: NodeJS.ProcessEnv
 
+beforeAll(() => {
+  oldEnv = JSON.parse(JSON.stringify(process.env))
+  process.env.ETHEREUM_RPC_URL = 'FAKE_ETHEREUM_RPC_URL'
+  process.env.CACHE_ENABLED = 'false'
+})
+
+afterAll(() => {
+  process.env = oldEnv
+})
+
```

Comment:
Was this change intentional for this PR?

Reply:
Yes, this test will error out with this PR if this isn't changed :sweat_smile:

Reply:
Yeah I see why it might cause an error without this fix...